gini/gini-vision-lib-android
# Updated the layout of the Analysis Screen to be able to show suggesti…

Updated the layout of the Analysis Screen to be able to show suggestions.
Now it looks like the designs, however the sizes of the text and opacity of the background overlay is not 100% decided I guess.

### How to test
best check in the layout editor the preview since nothing is set yet only the tools attribute is used.

### Merging
none
